package.json

```diff
@@ -9,15 +9,16 @@
   "homepage": "https://RoundingWell.com",
   "scripts": {
     "analyze": "cross-env NODE_ENV=production webpack --profile --json > webpack.stats.json && webpack-bundle-analyzer webpack.stats.json",
-    "build": "cross-env NODE_ENV=production webpack",
-    "build:ci": "cross-env NODE_ENV=test webpack",
+    "build": "npm run fixtures:generate && cross-env NODE_ENV=production webpack",
```

Comment:
I don't think this should be tied to the build.  It's only related to Cypress at the moment, and may always be that way.  So it'd be better to call it prior to calling Cypress..  In fact, might be able to just make a Cypress plugin: https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/plugins/state-colors.js
https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/plugins/index.js#L19

And in that case all the `faker-generator.js` code could live over there..